rnsdomains/rns-manager-react
# [DRAFT] Set resolution when creating a subdomain

**This is in draft status until the issue in the JS library is resolved. **

## To test

- start with a domain
- register a subdomain without checking the resolution
- **expect**: subdomain to be setup without RSK resolution
- register a subdomain with owner === sender and resolution ticked
- **expect**: 2 transactions
    - navigate to /resolve and check RSK resolution setup
    - log into subdomain and check RSK resolution setup
- register a subdomain with owner as someone else, resolution ticket
- **expect**: 3 traansactions
    - navigate to /resolve and check RSK resolution setup
    - log into subdomain and check RSK resolution setup


Closes #267
